src/components/Home.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div>
+    <tool-bar/>
+    <img alt="Arara" src="../assets/ArBC_logo.png" width="640" height="410"/>
+    <div><br><br><br></div>
+    
+    <md-card md-with-hover>
+      <md-ripple>
+        <md-card-header>
+          <md-icon class="md-size-4x" :md-src="require('@/assets/github.svg')"/>
+        </md-card-header>
+
+        <md-card-content>
+          Para ter acesso ao github do projeto clique no botâo abaixo.
+          <br><br><br>
+        </md-card-content>
+
+        <md-card-actions>
+          <md-button 
+            class="md-icon-button md-fab" 
+            target="_blank"
+            href="https://github.com/fga-eps-mds/2019.2-ArBC"
+          >
+            <md-icon :md-src="require('@/assets/github.svg')"/>
+          </md-button>
+        </md-card-actions>
+      </md-ripple>
+    </md-card>
+
+    <md-card md-with-hover>
+      <md-ripple>
+        <md-card-header>
+          <md-icon class="md-size-4x">camera_alt</md-icon>
+        </md-card-header>
+
+        <md-card-content>
+          Para ter acesso a nossa ferramenta de auxilio à alfabetização de crianças clique no botão abaixo.
+          <br><br>
+        </md-card-content>
+
+        <md-card-actions>
+          <md-button 
+            class="md-icon-button md-fab" 
+            target="_blank"
+            href="./camera"
+          >
+            <md-icon >camera_alt</md-icon>
+          </md-button>
+        </md-card-actions>
+      </md-ripple>
+    </md-card>
+
+    <md-card md-with-hover>
+      <md-ripple>
+        <md-card-header>
+          <md-icon class="md-size-4x">help</md-icon>
+        </md-card-header>
+
+        <md-card-content>
+          Para ter acesso a wiki do projeto, que contém diversas informações úteis sobre os participantes e o software, clique no botão abaixo.
+        </md-card-content>
+
+        <md-card-actions>
+          <md-button 
+            class="md-icon-button md-fab" 
+            target="_blank"
+            href="https://jlucassr.github.io/ArBC-Pages/"
+          >
+            <md-icon >help</md-icon>
+          </md-button>
+        </md-card-actions>
+      </md-ripple>
+    </md-card>
```

Comment:
Ao invés de fazer um card pra cada coisa, cria um componente de Card e passa os dados que mudam como props pra ele. Aí tu chama esse componente as vezes que for necessário. O código fica mais separado e organizado

Reply:
Corrigido!